src/Simplex/Chat/Types.hs

```diff
@@ -825,7 +825,14 @@ data GroupMember = GroupMember
     -- but it's correctly set on read (see toGroupInfo)
     memberChatVRange :: VersionRangeChat,
     createdAt :: UTCTime,
-    updatedAt :: UTCTime
+    updatedAt :: UTCTime,
+    supportChat :: Maybe GroupMemberSupportChat
+  }
+  deriving (Eq, Show)
+
+data GroupMemberSupportChat = GroupMemberSupportChat
+  { chatTs :: UTCTime,
+    unanswered :: Bool
```

Comment:
based on the discussion, also needs mentions and unread counts

Reply:
Mentions are actually important, as one admin can comment on the question and effectively ask that another admin answers.

Reply:
tbc maintain in store or calculate on read